<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Emily C. Zabor</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">Research and publications</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="talks.html">Talks</a>
</li>
<li>
  <a href="mailto:ecz2108@cumc.columbia.edu">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/zabore">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/zabormetrics">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/emily-zabor-59b902b7/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p><link rel="stylesheet" href="styles.css" type="text/css"></p>
<div id="jsm-2017" class="section level2">
<h2>JSM 2017</h2>
<p>At the Joint Statistical Meetings in Baltimore, MD, I presented a talk titled “Dimension reduction in the study of etiologic heterogeneity” on August 1, 2017. A PDF of the slides is available <a href="https://github.com/zabore/talk-slides/blob/master/2017_JSM_Zabor_dimension_reduction_etiologic_heterogeneity.pdf">here</a>.</p>
<script async class="speakerdeck-embed" data-id="abe8ee65f2c34eba9537c421600880d5" data-ratio="1.33333333333333" src="//speakerdeck.com/assets/embed.js"></script>
<p>Traditionally cancer epidemiologic research has been organized by the site of disease, so that, for example, a research may seek to identify risk factors for breast cancer as a whole. However with the rise of molecular and genomic profiling in recent decades, attention has become increasingly focused on identification of subtypes of disease. As a result, cancer epidemiologic research has shifted in focus to the search for risk factors that differ across subtypes of disease. The concept of differing risk factors according to subtypes of disease is known as etiologic heterogeneity.</p>
<p>A number of statistical methods are available to test for etiologic heterogeneity, and most rely on the use of polytomous logistic regression in the context of a case-control study to test for differences in relative risks according to an individual risk factor. However, these methods rely on a small number of tumor markers that are combined to form predefined subtypes. We have rather sought to develop an approach to search for etiologicallly distinct subtypes in the context of high dimensional tumor marker data. To illustrate the analytic strategy, imagine the situation where there are 3 underlying disease subtypes. Cases have data available on tumor markers, <span class="math inline">\(y_i\)</span>, and both cases and controls have data available on risk factors, <span class="math inline">\(x_i\)</span>. A polytomous logistic regression model is fit for a set of candidate subtypes, and predicted risks, <span class="math inline">\(r_ji\)</span> are obtained for each subtype j. More details follow about how candidate subtypes are identified.</p>
<p>We measure risk heterogeneity using the subtype-specific coefficients of variation, the covariances of predicted risks of individuals in the population, and the subtype-specific prevalences.</p>
<p>The first part of the equation represents the absolute explainable risk variation, defined by the coefficients of variation, <span class="math inline">\(K_j^2\)</span>, and the prevalences, <span class="math inline">\(\pi_j^2\)</span>, for each subtype j.</p>
<p>The second part of the equation represents the heterogeneity of risk profiles, where a low covariance term corresponds to a pair of discordant subtypes and conversely a high covariance term corresponds to a pair of concordant subtypes.</p>
<p>Thus we can construct a constant term <span class="math inline">\(D\)</span> that represents the absolute amount of variation explained by the risk factors, which we seek to maximize.</p>
<p>We have proposed to conduct an analysis of this type in three steps. First we perform k-means clustering with many random starts to identify a set of candidate subtypes. Next we calculate D for each set of candidate subtypes. So for example if we use 1000 random starts we will obtain 1000 D metrics. Finally, we select the “optimal” class solution as the one that maximizes D. However this approach is somewhat <em>ad hoc</em> in nature and to date we have not evaluated how sensitive the approach is to clustering method used.</p>
<p>Here I use a data example to assess the sensitivity of results to different clustering approaches. First I compare two unsupervised clustering approaches, k-means clustering and hierarchical clustering. With hierarchical clustering you can use different distance metrics and different agglomeration algorithms, which is the way the method decides to combine individuals into groups. Next we compare clustering on the full gene set versus doing up front filtering of the gene set using an unsupervised approach and versus doing up front filtering of the gene set using a supervised approach.</p>
<p>The data for this example study come from the Cancer and Steroid Hormone breast cancer case-control study. There were 2990 controls frequency matched to 551 cases. Both cases and controls had available risk factor data on common breast cancer risk factors. Breast cancer cases additionally had data avialable on 202 gene expression values. Gene expression data is naturally continuous, but we compare results with dichotomized gene expression data as well.</p>
<p>For simplicity throughout I focus on a 4-class solution, though selection of the correct number of classes is another challenge in a clustering analysis, and is an active area of research that I will not discuss today. Comparing k-means clustering to hierarchical clustering approaches, we find that hierarchical clustering results in very unbalanced average class size, across 1000 random starts, so that polytomous logistic regresssion could not even usually be applied. As a result, we focus solely on k-means clustering, which results on average in more balanced class size.</p>
<p>We compare clutering on the full gene set to clustering on principal components, and unsupervised dimension reduction approach, to clustering on a gene set pre-filtered according to univariate D for each gene, and finally to clustering on a gene set pre-filtered according to an F-statistic proposed by Zapala &amp; Schork, which I will go into more detail about in a few slides.</p>
<p>The x-axis shows the number of principal components and the y-axis shows the percentage of variation explained. Using continuous gene expression data, we identify a set of 46 principal components that explain 85% of the variation. Using binary gene expression data, we identify a set of 90 principal components that explain 85% of the variation.</p>
<p>For the supervised filtering approach using univariate D, we first calculate D for each binary gene expression value. Next we permute the class labels 10,000 times to get a reference distribution to calculate a p-value. The p-values are adjusted for multiple comparisons using the false discovery rate method. The gene index is on the x-axis and the <span class="math inline">\(log_{10}\)</span> p-value is on the y-axis. We identify a set of 43 genes with p &lt; 0.05.</p>
<p>Finally, an F-statistic resulting from a multivariate distance matrix regression is using for supervised filtering. A distance matrix formed by the gene expression values is the regression outcome and the risk factors of interest are the regression predictors. First we calculate the F-statistic for each gene individually and order them from largest to smallest F-statistic. Then we serially increase the gene set and each time a gene is added we recalculate the F-statistic. The gene set with the largest F-statistic is selected.</p>
<p>Using this approach we identify a 6-gene set using continuous data and a 2-gene set using binary data.</p>
<p>It is encouraging to note that all 6 of the continuous genes and both of the binary genes selected by the F-statistic are among the 43 genes selected by univariate D. If we look at the top 43 ranked genes according to the two supervised approaches, we find that 32 of the continuouos genes and 35 of the binary genes overlap.</p>
<p>Recall that we seek to maximize D. Using the full gene set, we get a D of 0.200 for the continuous data and 0.231 for the binary data. With the continuous data, we see that both of the supervised approaches result in a higher D than that of the full gene set whereas with the binary data, only the supervised approach using univariate D results in a higher D than that of the full gene set.</p>
<p>Comparing the optimal class solutions using the top 43 ranked continuous elements in each method, we find substantial agreement between the two supervised dimension reduction approaches and moderate agreement between the unsupervised and supervised approaches, as indicated by the numbers on the diagonal and the kappa statistics.</p>
<p>A similar pattern of results is seen for the binary data.</p>
<p>In conclusion, we can identify more strong etiologically distinct subtypes using supervised dimension reduction. However, the results of this data analysis must be interpreted cautiously, as the gold standard class solution is unknown. The F-statistic is desirable as compared to univariate D because of its computational simplicity. Simulation studies are underway to examine properties of these approaches in the context of a gold standard solution.</p>
<p>Many thanks to my dissertation advisors, Colin Begg and Shuang Wang.</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
